libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -276,6 +289,22 @@ public void setContent(CharSequence text) {
         mWPAndroidGlueCode.setContent(postContent);
     }
 
+    public void onToggleHtmlMode() {
+        if (!isAdded()) {
+            return;
+        }
+
+        toggleHtmlMode();
+    }
+
+    private void toggleHtmlMode() {
+        mEditorFragmentListener.onTrackableEvent(TrackableEvent.HTML_BUTTON_TAPPED);
+        mEditorFragmentListener.onHtmlModeToggledInToolbar();
+
+        mHtmlModeEnabled = !mHtmlModeEnabled;
```

Comment:
nitpick: wondering whether we should move this line to the top of the method: this way listeners and tracking happen after the fact (when we're sure we're past that point)

Reply:
Even though there is no need yet for checking this flag outside the fragment, makes sense to move it up. Done with 7d916b3786.